client/js/render.js

```diff
@@ -0,0 +1,192 @@
+"use strict";
```

Comment:
It doesn't look like "use strict" is one of the options there, but possibly. Let's not do this in this PR anyway, we can do that in another.